test/integration/content-sidebar/ContentSidebar.e2e.test.js

```diff
@@ -96,4 +97,57 @@ describe('ContentSidebar', () => {
             cy.get('@versionHistory').should('not.exist');
         });
     });
+
+    describe('activity feed comments', () => {
+        const getDraftJSEditor = () => cy.get('[contenteditable]');
+        const getTooltip = () => cy.get('[role="tooltip"]');
```

Comment:
Is it worth scoping these down to the comment components?

Reply:
I scoped down the `getDraftJSEditor` but tooltip I've left for now since Tether just appends it to the body and I couldn't figure a good way to be more specific